aiogram/dispatcher/middlewares.py

```diff
@@ -96,7 +96,7 @@ def is_configured(self) -> bool:
         :param args:
         :return:
         """
-        handler_name = f"on_{action}"
+        handler_name = f"on_"+action
```

Comment:
Doesn't follow project code-style. plus should be separated by space with left and right parts of expression.